data/json/monsters/insect_spider.json

```diff
@@ -2057,8 +2057,30 @@
     "death_function": { "message": "The %s's body leaks acid.", "effect": { "id": "death_acid", "hit_self": true } },
     "harvest": "arachnid_acid",
     "dissect": "dissect_insect_sample_single",
+    "upgrades": { "half_life": 42, "into": "mon_ant_acid_soldier_mega" },
     "flags": [ "ACIDPROOF", "SHORTACIDTRAIL", "CLIMBS", "HEARS", "POISON", "SEES", "SMELLS", "PATH_AVOID_FIRE", "PATH_AVOID_FALL" ]
   },
+  {
+    "id": "mon_ant_acid_soldier_mega",
+    "type": "MONSTER",
+    "name": { "str": "super soldier acid ant" },
+    "description": "A cow-sized, woolly brown ant.  Corrosive liquid seeping from her huge mandibles.  Given the relative strength of ants to their body size, you don't doubt that this one could shove cars around or rip through solid walls.",
+    "copy-from": "mon_ant_acid_soldier",
+    "proportional": { "hp": 10, "speed": 0.25, "dodge": 0.5, "vision_day": 2 },
```

Comment:
1/4th speed needs to go if you want them not to be a laughingstock, and 10x HP is pretty overblown (consistent, yes, but not *good*). I'd go with higher armor and a HP in the 400ish range (if you bump the vision range of the base up they should swarm anyway).

Reply:
I noticed the issue with speed during testing but I shrugged it off thinking that if that's the case with normal supersoldiers then there probably was a _reason_ for them to be this slow. These proportionals are directly copied from the regular supersoldier but I do see why 10x the HP feels insane